### ISSUE TITLE
blueboxes.rktd.gz files are not added to blueboxes-cache

### ISSUE THREAD

**Issue by ohspite** [racket/scribble#57]
I don't know if this is normal or just a peculiarity of Debian or Ubuntu packaging, but in the `/usr/share/doc/racket` there are many `blueboxes.rktd.gz` files.  It seems like the compressed files are not being read when running `make-bluueboxes-cache`.  

I tested by copying `/usr/share/doc/racket/reference/blueboxes.rktd.gz` to my desktop, and running
``` racket
(define tag (xref-binding->definition-tag (load-collections-xref) #'dict-ref 0)) 
(fetch-blueboxes-strs tag #:blueboxes-cache 
    (make-blueboxes-cache #t #:blueboxes-dirs (list (string->path "/home/don/desktop/"))))
```
When the blueboxes file is compressed, this results in `#f`, but when the blueboxes file is uncompressed the result is what you would hope: 
```
'("procedure"
  "(dict-ref dict key [failure-result]) -> any"
  "  dict : dict?"
  "  key : any/c"
  "  failure-result : (failure-result/c any/c)"
  "                 = (lambda () (raise (make-exn:fail ....)))")
```

I came across this while trying to figure out why the xrepl `,describe` command wasn't showing the documentation for many commands.

**Comment by rfindler** [comment]
I think the problem is that they were gzipped. How did that happen and why
would someone gzip random parts of our distribution?

**Comment by ohspite** [comment]
I just double checked to make sure it also occurs in the default repositories because I was using the Racket team's ppa (https://launchpad.net/~plt/+archive/ubuntu/racket).  But yes, it seems like the files are compressed in the official repositories as well.  This is all on Ubuntu 16.04.  Maybe @takikawa would know what's happening during packaging.

**Comment by takikawa** [comment]
Thanks for the report. This probably means that the Debian packaging (and Ubuntu) should use `DEB_DH_COMPRESS_ARGS` or `DEB_COMPRESS_EXCLUDE` in CDBS to specify that blueboxes should never be compressed.

(I can't look into this immediately, but I'll try to fix it and submit a fix upstream to Debian)

**Comment by rfindler** [comment]
Are they any files that are part of our distribution that can be randomly
compressed?!?

**Comment by takikawa** [comment]
This bug should now be fixed in the latest version of the PPA (for v6.7) which is now up. I've submitted the patch upstream to the Debian maintainer so it should get fixed there too.
